opentelemetry-api/java-agent/pom.xml

```diff
@@ -17,6 +17,7 @@
       <module>extensions</module>
       <module>TelemetryFilteredBaseOnRequestDuration</module>
       <module>extensions/FilterSpanBasedOnDuration</module>
+        <module>TestAzureCore</module>
```

Comment:
maybe better name?
```suggestion
        <module>AzureHttpClient</module>
```

Reply:
will rename it. i will add a README too.